2.0/README.md

```diff
@@ -0,0 +1,137 @@
+Ruby for general usage and OpenShift - Docker images
```

Comment:
Since I'm taking the https://github.com/openshift/mysql/pull/99 as a precedes could you please update the headers of all the READMEs in all your PRs to match the one from https://github.com/openshift/mysql/pull/99/files, to be consistent ?